wiki/versions/1.19/item/simple_item/groovy/src/main/groovy/org/quiltmc/wiki/simple_item/SimpleItemExample.groovy

```diff
@@ -0,0 +1,23 @@
+package org.quiltmc.wiki.simple_item
+
+import net.minecraft.item.Item
+import net.minecraft.item.ItemGroup
+import net.minecraft.util.Identifier
+import net.minecraft.util.registry.Registry
+import org.quiltmc.loader.api.ModContainer
+import org.quiltmc.qsl.base.api.entrypoint.ModInitializer
+import org.quiltmc.qsl.item.setting.api.QuiltItemSettings
+
+class SimpleItemExample implements ModInitializer {
```

Comment:
This might be a good place to use a script entrypoint, if you'd like, though honestly neither is necessarily better

Reply:
To show off the feature, probably is since it's simple enough to expand upon.